libs/labelbox/src/labelbox/schema/project.py

```diff
@@ -145,8 +147,7 @@ def is_chat_evaluation(self) -> bool:
         return self.media_type == MediaType.Conversational and self.editor_task_type == EditorTaskType.ModelChatEvaluation
 
     def is_auto_data_generation(self) -> bool:
-        return self.media_type == MediaType.LLMPromptCreation or self.media_type == MediaType.LLMPromptResponseCreation or self.is_chat_evaluation(
-        )
+        return (self.upload_type == UploadType.Auto) is True
```

Comment:
reduntant, return self.upload == UploadType.Auto

Also not sure if it's needed, maybe the field is good enough instead of needing a function

Reply:
![Screenshot 2024-07-01 at 7 36 41 PM](https://github.com/Labelbox/labelbox-python/assets/1299437/aa31951c-f5b1-432d-a1da-38ab62e9b8ce)
this is what you get if you try to do it directly... our `Field` stuff is not real Bool. Also the method help to wrap it all and since I already have it in place, makes it work better

Reply:
@adrian-chang I have updated as you suggested... this lint warning is invalid, I have verified in code self.upload_type is an Enum, not a Field